xml/System.Security.Cryptography/HashAlgorithm.xml

```diff
@@ -1333,11 +1342,11 @@ For more information about Dispose and Finalize, see [Cleaning Up Unmanaged Reso
         <Parameter Name="bytesWritten" Type="System.Int32" RefType="out" Index="2" FrameworkAlternate="netcore-2.1;netcore-2.2;netcore-3.0;netstandard-2.1" />
       </Parameters>
       <Docs>
-        <param name="source">To be added.</param>
-        <param name="destination">To be added.</param>
-        <param name="bytesWritten">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
+        <param name="source">The input to compute the hash code for.</param>
+        <param name="destination">The buffer to receive the hash value.</param>
+        <param name="bytesWritten">When this method returns, the total number of bytes written into <paramref name="destination"/>. This parameter is treated as uninitialized.</param>
+        <summary>Attempts to compute the hash value for the specified byte array.</summary>
+        <returns><see langword="true"/> if <paramref name="destination"/> is long enough to receive the hash value; otherwise, <see langword="false"/>.</returns>
```

Comment:
```suggestion
        <returns>
          <see langword="true"/> if <paramref name="destination"/> is long enough to receive the hash value; otherwise, <see langword="false"/>.</returns>
```